app/assets/javascripts/events.coffee

```diff
@@ -1,14 +1,16 @@
 $(document).ready ->
-  $('.events').infinitescroll(
+  $container = $('.events')
```

Comment:
maybe `$events` instead of `$container`? `container` can be confused with `.container` bootstrap css class. What do you think?